torchbench.py

```diff
@@ -823,6 +866,12 @@ def main():
                 experiment,
                 cos_similarity,
             )
+        stats_file = output_filename.split(".csv")[0] + "_stats.csv"
```

Comment:
@jansel This is the place, I am passing on a new filename. But, yeah, it is not as contained as it should be.